applications/luci-app-acme/htdocs/luci-static/resources/view/acme/acme.js

```diff
@@ -67,20 +86,69 @@ return view.extend({
 		o = s.taboption('general', form.Flag, "enabled", _("Enabled"));
 		o.rmempty = false;
 
+		o = s.taboption('general', form.ListValue, 'validation_method', _('Validation method'),
+			_('Standalone mode will use the built-in webserver of acme.sh to issue a certificate. ' +
+				'Webroot mode will use an existing webserver to issue a certificate. ' +
+				'DNS mode will allow you to use the DNS API of your DNS provider to issue a certificate.') + '<br />' +
+			_('Validation via TLS ALPN') + ': ' + _('Validate via TLS port 443.') + '<br />' +
```

Comment:
It's a bit weird that the first three uses "will use" and the last one uses ":" to delimit the name and description; shouldn't we be consistent here?

Reply:
Alright, that's fair...